src/Home/HomePages.tsx

```diff
@@ -0,0 +1,73 @@
+import "../App.css"
+
+import LeftNavBar from '../components/LeftNavBar/LeftNavBar';
```

Comment:
in each directory inside components you should create index.ts file where you could export default your component, that will enable you to avoid double imports